<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Downscale red</string>
  </property>
  <widget class="QPushButton" name="pushButton_loadImage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load image...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loaded">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newWidth">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newHeight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_newWidth">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_newHeight">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New height</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_downscaleRed">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>110</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Downscale red...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
